app/src/test/java/com/example/benchproject/app/movie/details/viewModel/MovieDetailsViewModelTest.kt

```diff
@@ -49,16 +51,28 @@ internal class MovieDetailsViewModelTest {
         "2022-02-10",
         "https://image.tmdb.org/t/p/w500/something1.com"
     )
+    private val movieId = 54431
     private val getMovieDetailsUseCase: GetMovieDetailsUseCase = mockk()
+    private val fragmentArgs = spyk(MovieDetailsFragmentArgs(movieId))
```

Comment:
Why it's a `spyk`? :)

Reply:
In eariler solution I used :
`val state = SavedStateHandle().apply { set("movieId", movieId) }` 
but Borys instructed me to mock xxxFragmentArgs. I tried some way to do this and `spyk` works for me if it is not a good practice in this case I can try find another way :D

Reply:
Ok, I understand and I deleted `spyk` now I just use `private val fragmentArgs = MovieDetailsFragmentArgs(movieId)`